contracts/burner/Cargo.toml

```diff
@@ -22,12 +22,13 @@ incremental = false
 overflow-checks = true
 
 [features]
+# Change this to [] if you don't need Windows support and want faster integration tests.
```

Comment:
Hmmm.. wasted some time reflecting on Cargo files.

There is eg:

```toml
[target.x86_64-pc-windows-gnu.dependencies]
byte = "0.2.4"
```

But I cannot find the equivalent for enabling target-specific feature flags.

Minor point, but it would good to set a proper template for all the contracts

Reply:
This is only relevant for integration tests. Unit tests are not affected.

> The alternative would be default [], cargo test does the right thing on osx/linux, and you would need cargo test --features cranelift to run on windows at all.

Jupp, but it also requires different documentation. I went for the less optimized but more beginner friendly way. But open for change.

> Hmmm.. wasted some time reflecting on Cargo files.

See also https://github.com/CosmWasm/cosmwasm/issues/649, which is probably what you are looking for